ms-oidc/src/main/java/net/trajano/ms/oidc/internal/ServiceConfiguration.java

```diff
@@ -65,15 +63,13 @@ public URI getRedirectUri() {
     @PostConstruct
     public void init() throws IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)